mods/ra/maps/monster-tank-madness/map.yaml

```diff
@@ -2115,7 +2115,7 @@ Rules:
 			GenericVisibility: Enemy
 			ShowOwnerRow: false
 		AnnounceOnSeen:
-	^Infantry:
+	^Soldier:
```

Comment:
This doesn't look correct to me.
You're showing the `OwnerRow` of civilians etc now?